hooks/playbooks/tempest_store_timing_data.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Store stestr timing data in internal repository
+  hosts: localhost
+  gather_facts: false
+  vars:
+    repo_path: "/tmp/timing-data"
+    tests_log_dir: "{{ ansible_user_dir }}/ci-framework-data/tests/test_operator"
+    timing_data_repo: "https://user:{{ git_token }}@gitlab.cee.redhat.com/ci-framework/timing-data"
```

Comment:
let's keep `https://repo.example.org/project/timing-data` here and the full desired value pass just as variable when the hook is called